addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_image.py

```diff
@@ -261,5 +261,8 @@ def __get_tex_from_socket(blender_shader_socket: bpy.types.NodeSocket, export_se
 def __is_blender_image_a_jpeg(image: bpy.types.Image) -> bool:
     if image.source != 'FILE':
         return False
-    path = image.filepath_raw.lower()
-    return path.endswith('.jpg') or path.endswith('.jpeg') or path.endswith('.jpe')
+    if image.filepath_raw == '':
```

Comment:
Think you need `and image.packed_file`. Otherwise it would throw when you try to do `.data` on None.